curriculum/challenges/english/07-scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/65687b182c2a8fb1470e0b97.md

```diff
@@ -9,6 +9,8 @@ dashedName: step-20
 
 Within the `for` loop, use the `+=` operator to add the `digit` to the `sum_of_odd_digits` variable.
 
+Doing this your script throws a TypeError because you are trying to add a string to an integer, but don't worry, you will learn more about how to make it work in the next step.
```

Comment:
```suggestion
Doing this your script throws a `TypeError` because you are trying to add a string to an integer, but don't worry, you will learn more about how to make it work in the next step.
```